addons/sale_service/models/sale_order_line.py

```diff
@@ -34,6 +34,11 @@ def _auto_init(self):
             """)
         return super()._auto_init()
 
+    def _get_partner_display(self):
+        if self.env.context.get('task_view', False) and self.order_partner_id.parent_id:
+            return self.order_partner_id.parent_id.ref or self.order_partner_id.parent_id.name
+        return super()._get_partner_display()
```

Comment:
I don't think it makes sense to change this just in some views, that's inconsistent and confusing. That'll be something to see with AJU and DALA.

Reply:
they've discussed about it, and agreed to simply add the change to the generic name get of sols, not just for task views.